.travis.yml

```
@@ -4,14 +4,22 @@ sudo: required
 
 matrix:
   include:
-  - name: "services contracts & digital-twin tests"
+  - name: "services contracts"
```

```suggestion
  - name: "service contracts"
```
Or was this name intentional? ^^